apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -435,12 +436,17 @@ export default function SectionsSetUpContainer({
         isNewSection={isNewSection}
       />
 
-      <CurriculumQuickAssign
-        id="uitest-curriculum-quick-assign"
-        isNewSection={isNewSection}
-        updateSection={(key, val) => updateSection(0, key, val)}
-        sectionCourse={sections[0].course || consolidatedCourseData()}
-        initialParticipantType={sections[0].participantType}
+      {/* Allow the curriculum quick assign region to be configured per-region */}
+      <GlobalEditionWrapper
+        component={CurriculumQuickAssign}
+        componentId="CurriculumQuickAssign"
+        props={{
+          id: 'uitest-curriculum-quick-assign',
+          isNewSection: isNewSection,
+          updateSection: (key, val) => updateSection(0, key, val),
+          sectionCourse: sections[0].course || consolidatedCourseData(),
+          initialParticipantType: sections[0].participantType,
```

Comment:
We want to be able to add a property to add a filter for the courses per region. Hence, it gets wrapped in a region wrapper.